components/Header/Header.test.js

```diff
@@ -0,0 +1,11 @@
+import "@testing-library/jest-dom";
+import { render, screen } from "@testing-library/react";
+import Header from "./Header";
+
+test("Displays me a Header", () => {
```

Comment:
I'm enjoying the perspective, but as long as you're not a Header, you might not want to talk in the first person 😄

Reply:
Oh, I just noticed that I misread the description. I read "Displays me as a Header". That's why I mentioned the first-person perspective, lol. Nevertheless, if you drop the "me" the description reads more like an English sentence.

Reply:
Yeah it's fine, i know what you mean :) I replaced the "me" with an "a".

What if i'm not really creative btw? It's really hard for me to get into the testing mode right know :/

Reply:
Take your time then. Look into projects of your fellow developers. It might inspire you.

Reply:
@Kheirah Well, i took my time. It wasn't fun, but i added a testing form because i can't use my form inside the Map component. I hope it was enough creativity for you :D